CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Updates Bootstrap generator for supporting Bootstrap 5 with popper.js. ([@abhaynikam][])
 * Adds Overcommit generator for RuboCop. ([@coolprobn][])
 * Updates RuboCop generator to add rubocop-rake extension and support test frameworks ([@coolprobn][])
+* Adds Letter Opener generator. ([@coolprobn][])
```

Comment:
@coolprobn I've released new version of the gem(v0.12). can you move this changelog entry to unreleased section?